lib/cartopy/io/shapereader.py

```diff
@@ -317,8 +316,8 @@ def zip_file_contents(self, format_dict):
 
         """
         for ext in ['.shp', '.dbf', '.shx', '.prj', '.cpg']:
-            yield ('ne_{resolution}_{name}'
-                   '{extension}'.format(extension=ext, **format_dict))
+            yield Path('ne_{resolution}_{name}'
+                       '{extension}'.format(extension=ext, **format_dict))
```

Comment:
Not sure if we should change these, as `zip_file_contents` is technically public and could be overridden?

Reply:
OK, I've removed that change now as well and left it returning a string and reworked the path handling for later.